### DISCUSSION TITLE
BLAKE2 Replacement

### DISCUSSION THREAD

**Discussion by valeriob01** [preda/gpuowl#228] [Ideas]
Soon BLAKE3 1.0 will be released and it will be much faster than BLAKE2.
Ref.: https://github.com/BLAKE3-team/BLAKE3

**Comment by valeriob01** [comment]
The BLAKE3 release 1.0.0 is out

https://github.com/BLAKE3-team/BLAKE3/releases/tag/1.0.0

**Comment by selroc** [comment]
BLAKE3 release 1.2.0 is out

https://github.com/BLAKE3-team/BLAKE3/releases/tag/1.2.0
